### ISSUE TITLE
FAQ: Can I run a lot of Pocket Nodes in front of a cluster of external blockchain nodes?

### ISSUE THREAD

**Issue by jeremybeal11** [pokt-network/pocket-core#1113]
### PURPOSE
This issue is meant to serve as a reference to whether you can run multiple pocket nodes in front of a cluster. 
### GUIDELINES
- If FAQ does not comprehensively cover the issue, community comments and contributions are welcome
- Although the issue is closed, FAQ issues are meant to be maintained and edited over time
## Can I run a lot of Pocket Nodes in front of a cluster of external blockchain nodes?
In short, yes. This in fact is the most profitable way to participate in the network.

### NOTES
N/A
